docs/application/dotnet/guides/security/privacy-related-permissions.md

```diff
@@ -31,7 +36,7 @@ To enable your application to use the privacy-related permissions functionality:
 
     It means that the methods can be employed in any UI event handler (such as button click, timer event, system event, and application state change event). If you want to resolve privileges during application startup, call these methods from the Xamarin.Forms resume and start life-cycle methods (`Xamarin.Forms.Application.OnResume()` and `Xamarin.Forms.Application.OnStart()`).
 
-    > **Note**
+    > [!NOTE]
     >
     > The `Tizen.Security.PrivacyPrivilegeManager` class is not thread-safe.
```

Comment:
@yl33 I guess you might have missed editing note styling here.

Reply:
Right. Removed that one, too.